lib/longestString.js

```diff
@@ -0,0 +1,18 @@
+function longestString(array){
+
+  var longest_word = "";
+  if (array.length === 0) {
+    return null;
+  }
+  else{
+
+  array.forEach(function(word) {
```

Comment:
watch out for indenting, I noticed that somewhere else as well. Since this is inside a new block `else` it would be indented